app/src/main/java/ch/epfl/qedit/view/quiz/QuestionFragment.java

```diff
@@ -44,12 +44,7 @@ public View onCreateView(
                 .getFocusedQuestion()
                 .observe(
                         getViewLifecycleOwner(),
-                        new Observer<Integer>() {
```

Comment:
I like when your replace those ugly anonymous classes, thank you !

Reply:
Oh yes, I missed this one, thx!